i18n/en.json

```diff
@@ -5,19 +5,23 @@
 	"whowrotethat-activation-link": "Who Wrote That?",
 	"whowrotethat-activation-link-tooltip": "Activate Who Wrote That?",
 	"whowrotethat-deactivation-link": "Close Who Wrote That?",
-	"whowrotethat-state-pending": "<strong><em>Who Wrote That?</em></strong> is loading. This might take a while.",
+	"whowrotethat-state-pending": "$1 is loading. This might take a while.",
+	"whowrotethat-state-pending-name": "Who Wrote That?",
 	"whowrotethat-state-error": "API Error: $1",
 	"whowrotethat-error-refresh": "Please refresh the page or try again later.",
 	"whowrotethat-error-later": "Please try again later.",
 	"whowrotethat-error-contact": "Please $1 about this issue.",
 	"whowrotethat-error-contact-link": "contact us",
 	"whowrotethat-ready-title": "Who Wrote That?",
 	"whowrotethat-ready-general": "Hover to see contributions by the same author. Click for more details.",
-	"whowrotethat-tour-welcome-title": "<strong><em>Who Wrote That?</em></strong> is enabled!",
+	"whowrotethat-tour-welcome-title": "$1 is enabled!",
+	"whowrotethat-tour-welcome-title-name": "Who Wrote That?",
 	"whowrotethat-tour-welcome-description": "Activate the tool from the sidebar. Hover over the text to see the author and their other contributions to this page. Click to get more details.",
 	"whowrotethat-tour-welcome-dismiss": "Got it!",
 	"whowrotethat-revision-added": "$1 added this on $2.",
-	"whowrotethat-revision-attribution": "They have written <strong>$1%</strong> of the page.",
-	"whowrotethat-revision-attribution-lessthan": "They have written <strong>less than 1%</strong> of the page.",
+	"whowrotethat-revision-attribution": "They have written $1 of the page.",
```

Comment:
I guess this isn't really the fault of this PR since it was there before, but "They" should be {{GENDER}} and we should send the username in for the gender. Maybe as a followup.

Reply:
Good catch. Will make another PR.